deploy/helm/sumologic/templates/logs/collector/otelwindows/service.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if and (eq (include "logs.collector.otelwindows.enabled" .) "true") (.Values.otelwindows.metrics.enabled) }}
```

Comment:
Can't we use the same Service, ServiceAccount, and similar resources for both Linux and Windows?